### ISSUE TITLE
Reduce input formatting requirements

### ISSUE THREAD

**Issue by zwsmith200** [pavanravindra/amino#9]
Currently, we expect the input trajectory to be formatted in a specific way. We should update this to be more user-friendly.

**Comment by k9cdt** [comment]
This is only a problem for this current CLI program. Suppose you import amino as a separate module. You can use the OrderParameter constructor to construct OP objects from any input source. 

See `reduce_features` method in https://github.com/tiwarylab/af2rave/blob/main/af2rave/features.py

**Comment by zwsmith200** [comment]
I agree that this only affects the standalone CLI. It's low priority but it does make AMINO a bit less user-friendly. Admittedly the expectation of PLUMED inputs may be more of a barrier to entry than the specific formatting so this is probably a small perturbation on ease of use. 

The issue is mostly to keep it on our radar after our discussion.
